telco-core/configuration/reference-crs-kube-compare/optional/networking/multus/bond_cni/bond_sriov.yaml

```diff
@@ -0,0 +1,36 @@
+# optional
+# count: 0-N
+---
+apiVersion: k8s.cni.cncf.io/v1
+kind: NetworkAttachmentDefinition
+metadata:
+  name: {{ .metadata.name }}
+  namespace: {{ .metadata.namespace }}
+spec:
+  config: {{ .spec.config | toJson }}
```

Comment:
I think technically you'd want to use "toYaml" instead (because you want to render the data as a yaml field), though I suppose since Yaml is a subset of Json, it'll be the same; preserving a string as a string with appropriate quoting.

Reply:
Needs to stay as json. We have the same in the generic NAD validation file.